samples/ingestion/ingestion-client/FetchTranscription/Startup.cs

```diff
@@ -28,6 +30,20 @@ public override void Configure(IFunctionsHostBuilder builder)
                 builder.Services.AddDbContext<IngestionClientDbContext>(
                   options => SqlServerDbContextOptionsExtensions.UseSqlServer(options, FetchTranscriptionEnvironmentVariables.DatabaseConnectionString));
             }
+
+            builder.Services.AddAzureClients(clientBuilder =>
+            {
+                clientBuilder.AddServiceBusClient(FetchTranscriptionEnvironmentVariables.StartTranscriptionServiceBusConnectionString)
```

Comment:
In future we probably should switch to use AzureDefaultCredential, instead of connection string.

Reply:
No need to handle it now. You can go ahead and merge

Reply:
yes you're right, there's already a gh issue for it:
https://github.com/Azure-Samples/cognitive-services-speech-sdk/issues/2457

Adding the services via DI will make it easier to switch to managed identity